menuxconfig.py

```diff
@@ -0,0 +1,765 @@
+        max_str_len = len(max(split_data, key=len))
+
+        max_y = 20 if num_lines > 20 else num_lines
+        min_y = 2 if max_y < 2 else max_y
+
+        max_x = 40 if max_str_len > 40 else max_str_len
+        min_x = 10 if max_x < 10 else max_x
+
+        return min_x, min_y, max_x, max_y
+
+    def __init__(self, root, title, help_dict):
+        top = self.top = tk.Toplevel(root)
+        min_size_w = 300
+        min_size_h = 300
+        for key in help_dict:
+            text_title = key
+            text_data = help_dict[key]
+            min_x, min_y, max_x, max_y = self.min_max(text_data)
+            min_size_w = min_size_w + min_x
+            min_size_h = min_size_h + min_y
+
+        top.wm_minsize(width=min_size_w, height=min_size_h)
+        top.title(title)
+
+        self.b = tk.Button(top, text='Ok', command=self.cleanup)
+        self.b.pack()
+
+        text_list = []
+        for key in help_dict:
+            text_title = key
+            text_data = help_dict[key]
+            min_x, min_y, max_x, max_y = self.min_max(text_data)
+
+            frame = DoubleScrollbarFrame(
+                top, relief='sunken', width=min_x, height=min_y)
+            subframe = ttk.Frame(frame.get_frame())
+            lbl = ttk.Label(subframe, text=text_title)
+            lbl.pack(anchor='center', fill=tk.Y, expand=tk.FALSE)
+            hlp = tk.Text(subframe, wrap='none', width=min_x, height=min_y)
+            hlp.insert(tk.END, text_data)
+            hlp.configure(state=tk.DISABLED)
+
+            frame.hscrollbar.config(command=hlp.xview)
+            frame.vscrollbar.config(command=hlp.yview)
+            hlp.config(yscrollcommand=frame.vscrollbar.set)
+            hlp.config(xscrollcommand=frame.hscrollbar.set)
+
+            hlp.pack(anchor='w', fill=tk.BOTH, expand=tk.TRUE)
+            subframe.pack(padx=15, pady=5, fill=tk.BOTH, expand=tk.TRUE)
+            frame.pack(padx=5, pady=5, fill=tk.BOTH, expand=tk.TRUE)
+
+    def cleanup(self):
+        self.top.destroy()
+
+
+class TextPopupWindow(object):
+
+    def __init__(self, root, config, value):
+        self.top = top = tk.Toplevel(root)
+        self.l = tk.Label(top, text=config)
+        self.l.grid(row=0, column=0)
+        self.e = tk.Entry(top)
+        self.e.delete(0, tk.END)
+        self.e.insert(0, value)
+        self.e.grid(row=0, column=1)
+        self.b = tk.Button(top, text='Submit', command=self.Submit)
+        self.b.grid(row=1, column=0)
+        self.c = tk.Button(top, text='Cancel', command=self.Cancel)
+        self.c.grid(row=1, column=1)
+        self.v = False
+        self.top.grid()
+        self.top.wm_title("Set " + config)
+
+    def Submit(self):
+        self.value = self.e.get()
+        self.v = True
+        self.top.destroy()
+
+    def Cancel(self):
+        self.v = False
+        self.top.destroy()
+
+
+if __name__ == "__main__":
+    main()
+
+# Format via !autopep8 -i -a %
+# vim: et:ts=4
```

Comment:
Not hardcore about it, but I prefer to not have editor modelines and stuff in files.